nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -12,13 +12,16 @@ To upgrade to the latest release, follow the [upgrade chapter](https://nixos.org
 
 In addition to numerous new and updated packages, this release has the following highlights:
 
+- default Nix: 2.11 -\> 2.13. This includes in particular a [small language change](https://github.com/NixOS/nix/issues/8259) altering the way floats are represented in `builtins.toJSON`.
+  See the release notes for [2.13](https://nixos.org/manual/nix/stable/release-notes/rl-2.13.html) and [2.14](https://nixos.org/manual/nix/stable/release-notes/rl-2.14.html) for more info.
```

Comment:
The 2.14 release notes are 404.

Reply:
We went with the unstable manual, but please remember to also update stable when you can.